server/budgettracker.data/Repositories/UserRepository.cs

```diff
@@ -71,7 +97,7 @@ public bool Register(User userModel, out IEnumerable<string> errors)
         {
             string encryptedPassword = _cryptor.Encrypt(userModel.Password, _gateKeeperConfig.EncryptionKey, _gateKeeperConfig.Salt);
             userModel.Password = encryptedPassword;
-            int numDuplicates = _dbContext.Users.Count(user => user.UserName == userModel.Username);
+            int numDuplicates = GetActiveUserFromDb().Count(user => user.UserName == userModel.Username);
```

Comment:
should you await this one just like you did for the one above (line 78)

Reply:
After looking at this, I realized I shouldn't have use out variables in the repository. This makes it so that method can't be async. Therefor, I can't await in this method. I created an issue to refactor it but for the sake of issue scope, let's just let it go here and we'll fix it in the refactor: #54